README.md

```diff
@@ -10,12 +10,10 @@ See [how to contribue](#how-to-contribute)
 
 ## Legends
 
-```
-🇬🇧 Bahasa Inggris
-🇮🇩 Bahasa Indonesia
-🌟 Dokumentasi Resmi dari [developer.android.com](developer.android.com)
```

Comment:
@inibukanadit Hey there, thanks for your contribution. Can you adjust your commit to remove the link instead? 

Also if you don't mind rebasing from `master` will save me some work 😬

Thank you